crates/router/src/connector/globalpay/transformers.rs

```diff
@@ -22,31 +23,212 @@ pub struct GlobalPayMeta {
+                api_models::payments::WalletData::PaypalRedirect(_) => Ok(Self {
+                    account_name,
+                    amount: Some(item.request.amount.to_string()),
+                    currency: item.request.currency.to_string(),
+                    reference: item.attempt_id.to_string(),
+                    country: item.get_billing_country()?,
+                    capture_mode: item.request.capture_method.map(|cap_mode| match cap_mode {
+                        enums::CaptureMethod::Manual => requests::CaptureMode::Later,
+                        _ => requests::CaptureMode::Auto,
+                    }),
+                    payment_method: requests::PaymentMethod {
+                        apm: Some(requests::Apm {
+                            provider: Some(requests::ApmProvider::Paypal),
+                        }),
+                        authentication: None,
+                        bank_transfer: None,
+                        card: None,
+                        digital_wallet: None,
+                        encryption: None,
+                        entry_mode: Default::default(),
+                        fingerprint_mode: None,
+                        first_name: None,
+                        id: None,
+                        last_name: None,
+                        name: None,
+                        narrative: None,
+                        storage_mode: None,
+                    },
+                    notifications: Some(requests::Notifications {
+                        return_url: item.router_return_url.clone(),
+                        challenge_return_url: None,
+                        decoupled_challenge_return_url: None,
+                        status_url: None,
+                        three_ds_method_return_url: None,
+                    }),
+                    authorization_mode: None,
+                    cashback_amount: None,
+                    channel: Default::default(),
+                    convenience_amount: None,
+                    currency_conversion: None,
+                    description: None,
+                    device: None,
+                    gratuity_amount: None,
+                    initiator: None,
+                    ip_address: None,
+                    language: None,
+                    lodging: None,
+                    order: None,
+                    payer_reference: None,
+                    site_reference: None,
+                    stored_credential: None,
+                    surcharge_amount: None,
+                    total_capture_count: None,
+                    globalpay_payments_request_type: None,
+                    user_reference: None,
                 }),
-                ..Default::default()
+                api_models::payments::WalletData::GooglePay(data) => {
+                    let wallet_data_token = data.tokenization_data.token.as_str();
+                    let token = serde_json::from_str(wallet_data_token)
```

Comment:
Move this to connector utils and use it like 'get_wallet_token_as_json()'. Take to_connector_meta() in utils for reference